android/WALT/app/src/main/java/org/chromium/latency/walt/AudioTest.java

```diff
@@ -375,6 +390,15 @@ public void run() {
             ));
 
             if (testStateListener != null) testStateListener.onTestPartialResult(latencyCb_ms);
+            if (traceLogger != null) {
+                traceLogger.log(tb + waltDevice.clock.baseTime,
+                        (long) (tc + waltDevice.clock.baseTime - remaining_us),
+                        "Beep-to-callback",
+                        "Bar starts when beep is played and ends when callback received");
```

Comment:
Bar starts at the last frame of the buffer and ends when callback is received. Change start marker to tb + remaining_us.